operator/internal/pkg/prepare/deploy.go

```diff
@@ -103,7 +111,7 @@ func GetInstallInfos(ctx context.Context, manifestObj *v1alpha1.Manifest, defaul
 }
 
 func getDestinationConfigAndClient(ctx context.Context, defaultClusterInfo types.ClusterInfo,
-	manifestObj *v1alpha1.Manifest, clusterCache types.ClusterInfoCache,
+	manifestObj *v1alpha1.Manifest, processorCache types.RendererCache, installTarget string,
```

Comment:
why is installTarget not of type InstallCfgType?

Reply:
the flag is types string and is passed as it is. Type conversion happens inside, but other conversion during the very beginning is also fine.

Reply:
why is the flag type string is my question basically? also why is it called installTarget?